index.d.ts

```diff
@@ -12,12 +12,20 @@ interface FontAwesomeIconProps {
   pulse?: boolean
   rotation?: 90 | 180 | 270 | '90' | '180' | '270'
   swapOpacity?: boolean
-  size?: 'lg' | 'xs' | 'sm' | '1x' | '2x' | '3x' | '4x' | '5x' | '6x' | '7x' | '8x' | '9x' | '10x'
+  size?: '2xs' | 'xs' | 'sm' | 'lg' | 'xl' | '2xl' | '1x' | '2x' | '3x' | '4x' | '5x' | '6x' | '7x' | '8x' | '9x' | '10x'
```

Comment:
Updates sync types file to match changes made under https://github.com/FortAwesome/vue-fontawesome/pull/366